lib/containers/container_images.pm

```diff
@@ -110,9 +119,15 @@ sub test_opensuse_based_image {
     die 'Argument $image not provided!'   unless $image;
     die 'Argument $runtime not provided!' unless $runtime;
 
-    my ($host_version,  $host_sp,  $host_id)  = get_os_release();
-    my ($image_version, $image_sp, $image_id) = get_os_release("$runtime run --entrypoint '' $image");
-
+    my ($host_version, $host_sp, $host_id) = get_os_release();
+    my ($image_version, $image_sp, $image_id);
+    $image = 'sle15-working-container' if $runtime =~ /buildah/;
+    if ($runtime =~ /docker/) {
```

Comment:
I don't get this condition. Doesn't buildah support `--entrypoint` parameter?

Reply:
no `--entrypoint` in buildah. it is qute different from docker

Reply:
I see. But please keep also `podman` there as we test also images with `ENTRYPOINT` which we don't wanna use in this case:
```suggestion
    if ($runtime =~ /docker|podman/) {
```

Reply:
done 946e17650